### ISSUE TITLE
Support More Workflows

### ISSUE THREAD

**Issue by CodyRay** [semdiffdotnet/semdiff#96]
Right now we are focused on the feature branch workflow, would allowing the user to select between workflows be a good idea? (it would be a good idea if we could use that info in a meaningful way to provide more accurate results)

I got this after reading about different workflows here: [https://www.atlassian.com/git/tutorials/comparing-workflows](https://www.atlassian.com/git/tutorials/comparing-workflows)
- Centralized (SVN-ish)
- Feature Branch
- Gitflow (develop branch)
- Forking (traditional GitHub/upstream)

**Comment by cartermp** [comment]
Yes, switching between different workflows would be critical.  More a post-v1 sort of thing, though.
